nflow-engine/src/test/resources/logback-test.xml

```diff
@@ -12,7 +12,7 @@
       <onMismatch>NEUTRAL</onMismatch>
     </filter>
     <encoder>
-      <pattern>%d{dd.MM.yyyy HH:mm:ss,SSS} %-5p [%t] %-30.30c - [user:%X{username}] %msg%n</pattern>
+      <pattern>%d %-5p [%t] %-30.30c - [user:%X{username}] %msg%n</pattern>
```

Comment:
Why these changes in version bump PR?

Reply:
Without it the tests fail because `SSS` is not a valid timezone. This one is mentioned in the comments

Reply:
ok